tests/Metadata/Resource/Factory/FileConfigurationMetadataTest.php

```diff
@@ -11,6 +11,7 @@
 
```

Comment:
Shouldn't this test be split be in 2 (XML test and YAML test, eventually sharing some code in an `abstract` test class).